src/DotNetLightning.Core/Utils/LNMoney.fs

```diff
@@ -50,10 +50,7 @@ type LNMoney = | LNMoney of int64 with
         LNMoney.MilliSatoshis(Checked.op_Multiply 1000L sats)
 
     static member inline Satoshis(sats) =
-        LNMoney.Satoshis(int64 sats)
-
-    static member Satoshis(sats: uint64) =
-        LNMoney.MilliSatoshis(Checked.op_Multiply 1000UL sats)
```

Comment:
is this change related to this PR? if I had to choose one overload to leave here, I would leave the one that has the type explicitly specified

Reply:
> is this change related to this PR?

No, I should create another PR.

>  if I had to choose one overload to leave here, I would leave the one that has the type explicitly specified

May I ask why?
I just deleted this because I'm having compile error, I thought it is good idea to have inline method instead of having overload for all integral types... But I found inline static methods are tricky and sometimes it complains that it cannot infer types.

I don't know which is better, to leave only inline method or the one explicitly specified.

Reply:
well let's move it to a new PR indeed, and we discuss there?